gandi_tf/main.py

```diff
@@ -87,33 +148,71 @@ def generate_tf(domain, entries):
         f.write("  type   = each.value.type\n")
         f.write("  values = each.value.values\n")
         f.write("}\n")
-
+    if subdir is True:
+        with click.open_file(filename_tfimport, "w") as fi:
+            for cmd in commands:
+                fi.write(f"{cmd}\n")
     return commands
 
 
 @click.command(no_args_is_help=True)
 @click.option("--version", help="Display version information", is_flag=True)
+@click.option("--organization_id", help="Gandi organization ID (passing an organization ID will limit query on domain own by organization)", default=None)
```

Comment:
Could you turn this to a multi option so people could append multiple `--organization-id` parameters to the command.

```suggestion
@click.option("--organization-id", help="To list domains only owned by this organization id", default=None)
```

Reply:
Hi @marcaurele,
organization_id can't be renamed organization-id because "-id" is interpreted has id python builtin.

[https://api.gandi.net/docs/reference/#Sharing-ID](https://api.gandi.net/docs/reference/#Sharing-ID) can be pass as param only has a string an not a list, building a multiple organization_id support will need more work (next release maybe)